agent/bench-scripts/postprocess/linpack-prepare-input-file

```diff
@@ -151,14 +151,19 @@ cp -v ${linpack_binary} "${output_dir}${linpack_binary_file}"
 echo
 file "${output_dir}${linpack_binary_file}"
 echo
-echo "binary=${linpack_binary}" > "${output_dir}${linpack_metadata_file}"
+
+echo "Creating linpack script file"
+echo
+echo "#!/bin/bash" > "${output_dir}${linpack_script_file}"
+
+echo "echo binary='${linpack_binary}' > '${linpack_metadata_file}'" > "${output_dir}${linpack_script_file}"
```

Comment:
The second `>` should probably be `>>` in line 159...otherwise, there's no point in having line 157, right?

Reply:
ack